.github/pull_request_template.md

```diff
@@ -1,29 +1,40 @@
-<!--
-## Tips for pull requests
-* use chat, discussions, etc... to refine an idea for big and/or impactful changes
-* open them relatively early in draft mode and get regular feedback
-* add notes for reviewers and yourself as you go, so it's easy to maintain context
-* refine the pull request message over time
--->
+## Summary
+* what changed and how?
+* why are we changing it?
 
-<!--
-## Finalizing a PR:
+## Details
+* info that couldn't fit into the summary
+* relevant details; tricky parts
+* anything else
 
-### title:
-* reads like a short changelog line
+Fixes #(issue)
```

Comment:
`Fixes <full issue url>` - easier to do (no need to edit out ID), easier to use later (no need to piece ID back together when viewing bors commits), looks identically in GitHub preview

Reply:
Let's change this and the contribution guide together, in a separate PR.

Reply:
Contribution guide already has this elaboration

> - If you want to use automatic "fixes" feature of the github, please supply
>  full URL in the body instead, it makes it much easier to copy-paste and
>  see (with pure hash you first need to understand whether change was in
>  ``issues/`` or ``pull/``, then open random PR/issue and replace hash
>  URL).